app/templates/components/nav-bar.hbs

```diff
@@ -46,7 +46,7 @@
       {{/if}}
       {{#if (not-eq routing.currentRouteName 'create')}}
         <div class="item">
```

Comment:
Sometimes, when you run `npm install` it might change the file. Just revert the changes to that file in git.